EpikCord/__init__.py

```diff
@@ -77,9 +77,9 @@
 
 """
 :license:
-Some parts of the code is sourced from discord.py
+Some parts of this code are sourced from discord.py
 The MIT License (MIT)
-Copyright © 2015-2021 Rapptz
+Copyright © 2015-2020 Rapptz
```

Comment:
?

Reply:
We're in 2022, you've managed to go back a year.